ddl/util/event.go

```diff
@@ -41,8 +41,10 @@ func (e *Event) String() string {
 		}
 		ret += fmt.Sprintf(", Partition IDs: %v", ids)
 	}
-	if e.ColumnInfo != nil {
-		ret += fmt.Sprintf(", Column ID: %d, Column Name %s", e.ColumnInfo.ID, e.ColumnInfo.Name)
+	for _, columnInfo := range e.ColumnInfos {
+		if columnInfo != nil {
```

Comment:
Could we remove this `if`？ It looks useless.

Reply:
OK, done.